### PR TITLE
PRLC100-304 Functional Test Cases

### PR DESCRIPTION
**Pull request by elizereyes123**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/test/end-to-end/tests/C100Rebuild-BaseFlow.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+Feature('C100 Rebuild - Base flow');
```
Comment (vivek-sekhar) [inline src/test/end-to-end/tests/C100Rebuild-BaseFlow.js:1]: Are we executing the test case for only one flow? We have 4 types of flow right?

File: src/test/end-to-end/config.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
-const defaultPassword = 'Nagoya0102';
+const defaultPassword = 'Adoption@1234';
```
Comment (balaganesanhmcts) [inline src/test/end-to-end/config.js:1]: Please check this password common for all other tests in citizen front end

File: src/test/end-to-end/codecept.conf.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 exports.config = {
-  tests: './tests/*.js',
+  tests: './tests/C100Rebuild-BaseFlow.js',
```
Comment (balaganesanhmcts) [inline src/test/end-to-end/codecept.conf.js:2]: it should run for all tests

File: src/test/end-to-end/contents/CheckYourAnswers-content.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
+module.exports = {
```
Comment (balaganesanhmcts) [inline src/test/end-to-end/contents/CheckYourAnswers-content.js:1]: is that file name is common for all the Check your answers screen

File: src/test/end-to-end/tests/C100Rebuild-BaseFlow.js (hunk 1 above)
Comment (elizereyes123) [reply src/test/end-to-end/tests/C100Rebuild-BaseFlow.js:1]: Due to time constraints, I have only done the base flow. I will have to do the rest 3 flows after New Years

File: src/test/end-to-end/contents/CheckYourAnswers-content.js (hunk 4 above)
Comment (elizereyes123) [reply src/test/end-to-end/contents/CheckYourAnswers-content.js:1]: This is for the content of Check your answers for C100 Rebuild only

**Pull request closed without merging**